app/controllers/errors_controller.rb

```diff
@@ -1,26 +1,25 @@
 # Provides web actions that represent errors. Rails' standard error pages are static HTML with inline CSS; by using
 # a custom error controller we get all the layouts and CSS.
 class ErrorsController < ApplicationController
-  def not_found
-    render status: 404
-  end
+  def error
+    @exception = request.env['action_dispatch.exception']
+    @status = ActionDispatch::ExceptionWrapper.new(request.env, @exception)&.status_code
+    views = {
```

Comment:
Do other error codes not get reported or just not get nice names?  (I don't know what should happen here; this partial enumeration just made me wonder about others.)

Reply:
Only the 500s get reported; 403, 404, 409, 422, and 500 get nice error pages because those are the errors that come up in regular use. Any other errors _shouldn't_ come up so I haven't given them pages (and if they do, they're more likely to have been generated by nginx, where I can't control what they look like in any case).